pages/styles/events.css

```diff
@@ -0,0 +1,132 @@
+@import "../../common/styles/media-queries.css";
+
+.image {
+  margin: 2rem;
+  max-width: 500px;
+}
+
+.meetupCard {
+  padding: 0;
+}
+
+.meetupCard > div {
+  padding: 2rem;
+}
```

Comment:
Do you think these style overrides should be applied to all FlatCards too? Compare result against Success Story on homepage or the component itself on Storybook. I feel like we can deliver something that works for all usages without any style overrides.